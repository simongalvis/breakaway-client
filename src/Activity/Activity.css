
.Activity{
    display:flex;
    text-align: left;
    justify-content: center;
}
.activityItem{
    
    
    border: 1px solid rgb(231, 231, 231);
    border-radius:7px;
    max-width: 550px;
    align-self: center; 
    width:100%;
    margin: 3px ;
    
    
}
 #user-name{
    align-self: flex-start;
    margin:1%;
    width:100%;
}
#activity-title{
    float: left;
    width:100%;
    margin:1%
} 
#stat-bar{
    flex-direction: row;
    display: inline-flex;
    margin:1%;
    width:100%;
}
#description-section{
    margin:1%;
    width:100%;
    color: rgb(168, 168, 168);
    font-size:14px;
}
p{
    margin-right: 21px;
}
.stat-bar-div{
    font-size:30px;
    font-weight: lighter;
    color: rgb(168, 168, 168);
}