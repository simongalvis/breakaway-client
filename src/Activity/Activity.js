import React from 'react';
import './Activity.css'

class activity extends React.Component{



    render(){
        return(
            <h2>activity</h2>
        )
    }
}

export default activity;