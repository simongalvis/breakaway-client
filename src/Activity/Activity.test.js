import React from 'react'
import ReactDOM from 'react-dom'
import Activity from './Activity'
import renderer from 'react-test-renderer';

describe(`Avtivity component`, () =>{
  it(`renders without crashing`, () =>{
    const div = document.createElement('div');
  
    ReactDOM.render(<Activity/>, div);
  
    ReactDOM.unmountComponentAtNode(div);
  })
  it(`renders UI as expected`, () =>{
    const tree = renderer
      .create(<Activity/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})
