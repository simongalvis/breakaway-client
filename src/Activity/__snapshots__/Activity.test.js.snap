// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avtivity component renders UI as expected 1`] = `
<div
  className="Activity"
>
  <div
    className="activityItem"
  >
    <h6
      id="user-name"
    />
    <h4
      id="activity-title"
    />
    <br />
    <div
      id="stat-bar"
    >
      <p>
        Distance 
        <br />
         
         mi
      </p>
      <p
        className="stat-bar-div"
      >
        |
      </p>
      <p>
        Pace
        <br />
         
        NaN
         /mi
      </p>
      <p
        className="stat-bar-div"
      >
        |
      </p>
      <p>
        Time 
        <br />
         
         min
      </p>
    </div>
    <p
      id="description-section"
    >
      Notes: 
    </p>
  </div>
</div>
`;
